datajoint/attribute_adapter.py

```diff
@@ -38,10 +39,11 @@ def get_adapter(context, adapter_name):
         raise DataJointError('Support for Adapted Attribute types is disabled.')
     adapter_name = adapter_name.lstrip('<').rstrip('>')
     try:
-        adapter = context[adapter_name]
+        adapter = (context[adapter_name] if adapter_name in context
+                   else type_plugins[adapter_name]['object'].load())
     except KeyError:
         raise DataJointError(
-            "Attribute adapter '{adapter_name}' is not defined.".format(adapter_name=adapter_name)) from None
```

Comment:
@ixcat Thanks for the review man but I think there may be a misunderstanding here.

The `None`'s have not been *removed* but the mechanism in which they are defined was modified with the `plugin` update. Have a look [here](https://github.com/guzman-raphael/datajoint-python/blob/0525a78a2ee9e8d9a9a2ac70a216dc1a71140fe4/datajoint/errors.py#L20-L24). Effectively, if a DataJointError is raised and it is *not* due to an unverified plugin, then it behaves as before with the pretty, styled error i.e. `None` is added. 

Otherwise, a `PluginWarning` warning is added to the stack as the cause (this is so we can establish a clear line of support). Removing the `None` on specific calls was necessary so that it could be handled as a policy throughout.

Reply:
i see thanks for clarifying, didn't take into account the full exception stack :+1: